docs/extensions/cross-chain-data/oracles.mdx

```diff
@@ -0,0 +1,53 @@
+---
+sidebar_position: 1
+sidebar_label: "Overview"
+id: oracles-overview
+title: Oracles
+description: Build custom event-driven oracles.
+slug: /extensions/oracles-overview
+---
+
+Kwil networks have the ability to come to consensus on external system events, and trigger subsequent state changes based on the result of those events. This allows for network changes to be automatically triggered by external systems, such as other blockchain networks.
+
+Example usecases for Kwil oracles include:
+
+- Giving users native network gas when they burn an ERC20 token on Ethereum.
+- Updating a validator's power when they stake an SPL token on Solana.
+- Automatically syncing data from another Kwil network.
+
+## Anatomy of an Oracle
+
+Oracles are made up of two distinct components: event listeners, and resolutions.
```

Comment:
I think the docs should be consistent on which order "event listeners" and "resolutions" are presented. Everywhere else, resolutions are presented first. But here, it logically makes sense to present event listeners first (because nodes must agree on events before voting on the resolution). Should we change the rest of the docs to present event listeners first?